codecov.yml

```diff
@@ -0,0 +1,5 @@
+coverage:
+  status:
+    project:
+      default:
+        informational: true
```

Comment:
@robinholzi I will only make `codecov/project` action informational: i.e. it shows information but will not block PR. Otherwise currently we cannot proceed.

Reply:
Let's hope that I've configured correctly and this time all checks can pass

Reply:
Makes sense, thanks!